ops/services/app_functions/report_stream_batched_publisher/functions/common/reportingHandlers.ts

```diff
@@ -0,0 +1,128 @@
+import { Context } from "@azure/functions";
+import { DequeuedMessageItem, QueueClient } from "@azure/storage-queue";
+
+import { SimpleReportTestEvent } from "../FHIRTestEventReporter/dataHandlers";
+// import { ENV, uploaderVersion } from "../config";
+import { ReportStreamResponse } from "./rs-response";
+import {
+  deleteSuccessfullyParsedMessages,
+  publishToQueue,
+  reportExceptions,
+} from "./queueHandlers";
+import * as appInsights from "applicationinsights";
+
+//const { REPORT_STREAM_TOKEN, REPORT_STREAM_URL } = ENV;
+
+const telemetry = appInsights.defaultClient;
+
+export async function reportTestEvents(results: SimpleReportTestEvent[]) {
```

Comment:
nit: I would name this something like `reportTestEventsToUniversalPipeline` to distinguish it from the existing ReportStream call, since they'll be used simultaneously for quite some time.